src/TestStack.White/InputDevices/IKeyboard.cs

```diff
@@ -1,13 +1,81 @@
+using TestStack.White.UIItems.Actions;
 using TestStack.White.WindowsAPI;
 
 namespace TestStack.White.InputDevices
 {
     public interface IKeyboard
     {
+        /// <summary>
+        /// Enter a string
+        /// </summary>
+        /// <param name="keysToType">String to enter</param>
         void Enter(string keysToType);
+
+        /// <summary>
+        /// Send a strig to a specific <see cref="IActionListener"/>
+        /// </summary>
+        /// <param name="keysToType">String to send</param>
+        /// <param name="actionListener"><see cref="IActionListener"/> to receive the string</param>
+        void Send(string keysToType, IActionListener actionListener);
```

Comment:
As said up ahead. It could make sense to allow customers to create their own Keyboard and/or Mouse implementations that adhere to the Interface and use the IActionListener in their own manner.